### PR TITLE
extension added

### PR DESCRIPTION
**Pull request by heimseiten**

### CODE REVIEW

**Comment by aschempp** [comment]
Please fix the linter issues to get this merged.

**Comment by heimseiten** [comment]
sadly it is not possible, because "TinySlider", "Tiny Slider" or "Tiny-Slider" do not work in the title :/

**Comment by fritzmg** [comment]
You need to add that to the default allow list.

**Comment by heimseiten** [comment]
Thanks fritzmg! That would be awesome, but I can't find a place to do this.

**Comment by fritzmg** [comment]
The file is located [here](https://github.com/contao/package-metadata/blob/main/linter/allowlists/default.txt). You need to edit that file in your PR and commit and push your changes.

**Comment by heimseiten** [comment]
Thanks again. I added "Tiny Slider" in the default.txt

**Comment by aschempp** [comment]
Please fix the remaining validation issues

**Comment by aschempp** [comment]
@heimseiten can you please fix the validation issues?

**Comment by heimseiten** [comment]
I would like to do that, but i don't know how. I think first the request for default.txt would have to be accepted to pass the spell checker, right?

**Comment by fritzmg** [comment]
@heimseiten the word "Tiny" needs to be added to the default allow list. You added "Tiny Slider", but that won't do anything as these are actually two words.

**Comment by heimseiten** [comment]
Thank you very much and please excuse me. I never had to deal with the spell checker before.

File: linter/allowlists/default.txt [hunk 1]
```diff
@@ -223,6 +225,7 @@ Source
 Spectre
 Spotify
 sql
+standard
```
Comment (fritzmg) [inline linter/allowlists/default.txt:228]: Why did you add this to the default allow list? It should be removed.

File: meta/heimseiten/contao-tiny-slider-bundle/de.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+de:
+    title: Tiny Slider
+    description: >
+        Ersetzt den standard Contao-Slider mit dem Tiny Slider und fügt Funktionen hinzu wie Fade, Ken-Burns-Effekt usw..
```
Comment (fritzmg) [inline meta/heimseiten/contao-tiny-slider-bundle/de.yml:4]:
```suggestion
        Ersetzt den Standard Contao-Slider mit dem Tiny Slider und fügt Funktionen hinzu wie Fade, Ken-Burns-Effekt usw..
```

There is no adjective called "standard" in German, is there? Afaik you always capitalise it, when using it this way.

**Review by fritzmg — CHANGES_REQUESTED** [review]

**Review by fritzmg — APPROVED** [review]

**Review by Toflar — APPROVED** [review]

**Pull request merged**